tools/rebuild_provision_request.rb

```diff
@@ -240,13 +240,13 @@ def perform
 
   def run_it
     print "\nRE Running the Rest API POST to request_id: #{@options[:request_id]}\n\n" unless @quiet
-    result = RestClient.post build_url, output.to_json
+    result = require 'rest_client'; RestClient.post build_url, output.to_json
```

Comment:
We include `rest-client` at the top of this file.  Is it ok to be confused here?